deploy/docker/docker-compose.yml

```diff
@@ -1,33 +1,69 @@
 version: '2'
 
 services:
-  node1:
-    image: sshd
+  nstat:
+    image: intracom/nstat
```

Comment:
we could set for every container the `container_name` attribute, e.g.:
`container_name:nstat`

In this way, we will refer in the docker commands to the containers directly with these names